src/main/java/frc/robot/subsystems/drivetrain/commands/OverpoweredDrive.java

```diff
@@ -0,0 +1,111 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.Constants;
+import frc.robot.Robot;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import java.util.function.DoubleSupplier;
+
+import static frc.robot.Constants.SwerveDrive.VELOCITY_MULTIPLIER;
+
+/*
+    OverpoweredDrive is an assisted driving command that provides a smoother driving experience.
+    It corrects heading drifting, slows acceleration, and waits for wheels to reach their desired rotations before speeding.
+ */
+public class OverpoweredDrive extends HolonomicDrive {
```

Comment:
This command is a copy of `DriveSlowAcceleration`, the only difference is that you use the the heading controller in the command, and in `DriveSlowAcceleration` we use the heading controller in the SwerveDrive